Source/FakeItEasy.Specs/LazySpecs.cs

```diff
@@ -23,19 +25,37 @@ public interface IFoo
             ILazyFactory fake,
             Lazy<IFoo> lazy)
         {
-            "establish"
+            "Given a fake"
                 .x(() => fake = A.Fake<ILazyFactory>());
 
-            "when calling a method that returns a lazy"
+            "When calling an unconfigured method that returns a lazy of a fakeable type"
                 .x(() => lazy = fake.Create());
 
-            "it should return a lazy"
+            "Then it should return a lazy"
                 .x(() => lazy.Should().NotBeNull());
 
-            "it should return a lazy whose value is a dummy"
+            "And the value of the lazy should be a dummy"
                 .x(() => lazy.Value.Should().Be(FooFactory.Instance));
         }
 
+        [Scenario]
+        public static void LazyReturnValueOfNonFakeableType(
+            ILazyFactory fake,
+            Lazy<Bar> lazy)
+        {
+            "Given a fake"
+                .x(() => fake = A.Fake<ILazyFactory>());
+
+            "When calling a method that returns a lazy of a non-fakeable type"

```

Comment:
Sorry, I saw this comment earlier and didn't get to the code. I figured you'd already pushed the new commit.
I think dummyable sounds great. Well, at least I think it conveys the intention. StyleCop may think differently.

Yes, English is… quite welcoming of words. You wouldn't see Académie française allowing such an uncouth word!